src/Sentry.DiagnosticSource/Internals/DiagnosticSource/SentryEFCoreListener.cs

```diff
@@ -125,41 +128,54 @@ public void OnNext(KeyValuePair<string, object?> value)
             try
             {
                 //Query compiler Span
-                if (value.Key == EFQueryStartCompiling || value.Key == EFQueryCompiling)
+                if (value.Key is EFQueryStartCompiling or EFQueryCompiling)
```

Comment:
Love the pattern matching.  Looking at all the if statements in the rest of this method, seems like we could make this even more readable with a switch block.